### ISSUE TITLE
Kola Island Terminal Sign in wrong location

### ISSUE THREAD

**Issue by caerfinon** [caerfinon/KerbinSideRemasteredGAP#69]
The KK TKA sign is in the wrong position for the terminal

**Comment by caerfinon** [comment]
Correct position should be

```
	Instances
	{
		UUID = 859b0417-355d-40f4-ae13-1e0322b32c6f
		CelestialBody = Kerbin
		RelativePosition = 18.6274204,2.60279799,-344.608307
		Orientation = -8.10126003E-05,86.0001678,0.000101897524
		isScanable = False
		ModelScale = 1
		VisibilityRange = 25000
		Group = KSRGAP-KNAS
		GrasColor = 0,0,0,0
	}
```

**Comment by caerfinon** [comment]
Closed with v2.1.4
